domains/kartikshriwas.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "kartikshriwas",
+           "email": "",
+           "discord": "756002965323645069",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.pJe5J41LBdrD2xDWudBiVp66kIxdQIj60sraae1BOQ-SQ27mGzfBzsGjxzPfvEFSOFZd2sceXvK87x15PvOXRtdYvfUKNAFmt-oxcbfAfSmaiH4YtSFe7re3F5QEDNfYvYH96QsMwHQoj2CP5-5CceqWULYFvkusOL_4ScO8yYYYCRrDCKCJ8nP80wUT2ltwAikLqY6iJTwJvWS1LqiInC1qAn8VfA5-0Z6pgXCL7sMIsLWO-GZKXOqGUE_b5zV8LOgRml_RQgOlCZ306TVpOE2eF7M37nZ3ETbY3OWQo125q43oiisPZGo89sfL3jRxzd4OpaC56sEvEjj5_eXE6w.Bo1EpPh6LJSpWzAWSiqEYA.IHIiJ7JdCyD6CYg9bjHxHuBTJPluhpe2PGzmCR8A9RPmRcCitK4H5bTCXCw1R3qw6gIy2IugE9-Bt5iHulfHLHgPOmeg5xN8SHo5ea-Z7U4.1LJXKyAg2vnLjmnCoxQvGQ"
+        },
+    
+        "record": {
+            "CNAME": "kartikshriwas.github.io"
```

Comment:
404 not found, please give a preview of your website.

Reply:
kartikshriwas.github.io/kartikshriwas
and soon adding  kartikshriwas.github.io/PRMITR